### DISCUSSION TITLE
Transfer Tab DOES NOT APPEAR in the Robot Window

### DISCUSSION THREAD

**Discussion by JorgeYama** [cyberbotics/webots#6303] [General]
I have made a simulation of the Robotis Op2 in the Webots app, and I wanted to transfer it to the real robot. However, Im trying to transfer it but when the Robot Window is showed it doesnot appear

**Comment by stefaniapedrazzi** [comment]
Hi,
which version of Webots are you using?
Note that the custom robot window for the Robotis OP2, including the Transfer functionality, has been removed in Webots R2023b.

**Comment by JorgeYama** [comment]
Hi
Im using the latest version
Do you know what version has the Transfer Tab?

Thanks for answering

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by stefaniapedrazzi** [comment]
The latest version with the transfer tab is R2023a.

**Comment by JorgeYama** [comment]
Thank you
You save me
It Will be awesone the dance of the Robot Darwin Op2

Enviado desde Correo<https://go.microsoft.com/fwlink/?LinkId=550986> para Windows

De: Stefania ***@***.***>
Enviado: viernes, 7 de julio de 2023 04:44 a. m.
Para: ***@***.***>
CC: JORGE EMANUEL YAMA ***@***.***>; ***@***.***>
Asunto: Re: [cyberbotics/webots] Transfer Tab DOES NOT APPEAR in the Robot Window (Discussion #6303)


The latest version with the transfer tab is R2023a.
